.github/workflows/buf.yaml

```
@@ -8,12 +8,21 @@ jobs:
       - uses: actions/checkout@v3
       - uses: bufbuild/buf-setup-action@v1
       - uses: bufbuild/buf-lint-action@v1.0.3
+  breaking:
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```

We can change everything to run on latest now (which is 22.04 instead of 20.04).